study.md

```diff
@@ -93,15 +118,17 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.map { |array| array ** 2  }
 ```
 
 JavaScript:
 
 ```javascript
 const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
-// your answer here
+const squaredArray = array.map(function (array) {
+  return array ** 2
+})
 ```
```

Comment:
see other comment. 
`array.map(function(arrayElement){
    return arrayElement * 2
}`
you can call arrayElement anything you want, but just know it represents each element.